content/docs/5.1/deploy/benchmark/benchmark-instructions.md

```diff
@@ -0,0 +1,186 @@
+| PD            | 4 cores or above | 8 GB or above  | SAS, 200 GB+    | Gigabit LAN | 1                               |
+| Monitor       | 4 cores or above | 8 GB or above  | SAS, 200 GB+    | Gigabit LAN | 1 (can be the same as PD nodes) |
+| TiKV          | 8 cores or above | 32 GB or above | SSD, 200 GB+    | Gigabit LAN | 3                               |
+
+{{< info >}}
+Use local SSD instance store volumes. Local SSDs are low latency disks attached to each node, which maximizes performance. We do not recommend using network-attached block storage. We highly recommend you depoly TiKV over NVMe SSDs in order to maximizes the ability of it.
+{{< /info >}}
+
+2. Prepare services for the control node and component nodes.
+
+For the control node, we require:
+
+| package |    version     | comment             |
+| :------ | :------------: | :------------------ |
+| sshpass | 1.06 or above  | remote control      |
+| TiUP    | 0.6.2 or above | depoly TiKV cluster |
+
+For the component node:
+
+| package |     version     |          comment           |
+| :------ | :-------------: | :------------------------: |
+| sshpass |  1.06 or above  |       remote control       |
+| numa    | 2.0.12 or above | memory allocation strategy |
+| tar     | no requirements |           unzip            |
+
+For the YCSB node:
+
+| package                                       |     version     |  comment  |
+| :-------------------------------------------- | :-------------: | :-------: |
+| [go-ycsb](https://github.com/pingcap/go-ycsb) | no requirements | benchmark |
+
+
+{{< info >}}
+You can install [TiUP](https://github.com/pingcap/tiup) as described in [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+{{< /info >}}
+
+## Step 2. Deploy a cluster
+
+1. We could use the following topology to deploy our benchmark cluster via `tiup cluster`. **For more information, see [Production Deployment](../../install/production)**
+```yaml
+global:
+  user: "tikv"
+  ssh_port: 22
+  deploy_dir: "/tikv-deploy"
+  data_dir: "/tikv-data"
+server_configs: {}
+pd_servers:
+  - host: 10.0.1.1
+tikv_servers:
+  - host: 10.0.1.2
+  - host: 10.0.1.3
+  - host: 10.0.1.4
+monitoring_servers:
+  - host: 10.0.1.5
+grafana_servers:
+  - host: 10.0.1.6
+```
+
+2. Save the content above as `topology.yaml`:
+```sh
+tiup cluster deploy [cluster-name] [cluster-version] topology.yaml
+```
+
+3. Start the cluster:
+```sh
+tiup cluster start [cluster-name]
+```
+
+4. You can inspect the cluster information:
+```sh
+tiup cluster display [cluster-name]
+```
+
+## Step 3. Run a YCSB workload
+
+YCSB has six kinds of workloads whose main difference are the portion of different operations:
+
+1. Workload A: Update heavy workload
+2. Workload B: Read mostly workload
+3. Workload C: Read only
+4. Workload D: Read latest workload
+5. Workload E: Short ranges
+6. Workload F: Read-modify-write 
```

Comment:
```suggestion
6. Workload F: Read-Update
```

Reply:
The official description of workload F is `Read-modify-write`